front/components/assistant/conversation/input_bar/InputBar.tsx

```diff
@@ -166,18 +169,33 @@ export function AssistantInputBar({
       ...new Set(rawMentions.map((mention) => mention.id)),
     ].map((id) => ({ configurationId: id }));
 
-    onSubmit(
-      text,
-      mentions,
-      fileUploaderService.getFileBlobs().map((cf) => {
-        return {
-          title: cf.filename,
-          fileId: cf.fileId,
-        };
-      })
-    );
-    resetEditorText();
-    fileUploaderService.resetUpload();
+    let delayedLoading: number = 0;
+    // When we are creating a new conversation, we will disable the input bar, show a loading spinner and in case of error, re-enable the input bar
+    if (!conversationId) {
+      delayedLoading = _.delay(() => setLoading(true), 250);
```

Comment:
Why do we need a _.delay generally undesirable in front code no?

Reply:
To avoid showing the animation briefly if the create conversation is very fast (it felt glitchy).

Reply:
There will always be a request duration that will make it feel glitchy no?

Can we do a CSS transition instead?

Reply:
(there is actually one now already, wasn't the case at the beggining of this refacto, removing the delay)